sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -483,6 +485,34 @@ public CosmosQueryRequestOptions setThresholdForDiagnosticsOnTracer(Duration thr
         return this;
     }
 
+    /**
+     * Gets indexMetricsEnabled, which is used to obtain the index metrics to understand how the query engine used existing
+     * indexes and could use potential new indexes.
+     * The results will be displayed in QueryMetrics. Please note that this options will incurs overhead, so it should be
+     * enabled when debuging slow queries.
+     *
```

Comment:
```suggestion
     * By default the indexMetrics are disabled.
```

Reply:
I have that in line 494, to be consistent with isQueryMetricsEnabled().